src/Components/CategoriesPage.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import "../css/CatergoriesPage.css";
+
+const CategoriesPage = () => {
+  const [categories, selectCategories] = useState([]);
+  useEffect(() => {
+    fetch("https://fe-games.herokuapp.com/api/categories")
+      .then((res) => res.json())
+      .then((data) => {
+        selectCategories(data.categories);
+      });
```

Comment:
Lets extract this into an axios function in your api like we have with your others